CHANGELOG.next.asciidoc

```diff
@@ -44,6 +44,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Update Salesforce module to use new Salesforce input. {pull}37509[37509]
 - Tag events that come from a filestream in "take over" mode. {pull}39828[39828]
 - Fix high IO and handling of a corrupted registry log file. {pull}35893[35893]
+- Enable StatusReporter for filestream input. {pull}40121[40121]
```

Comment:
The `CHANGELOG.next.asciidoc` is the public facing changelog that goes into the release notes, we should write entries from the perspective of a user that does not know the internal details of our applications. A user looking to deploy Filebeat does not know what `StatusReporter` means.

You can phrase it by describing the observable behaviour, something like: *"Filebeat, when running under Elastic-Agent" reports detailed status for the Filestream input"*